WARNINGS.md

```diff
@@ -401,6 +402,17 @@ or at the beginning of a rule.
 
 --------------------------------------------------------------------------------
 
+## <a name="keyword-parameters"></a>Keyword parameter should be positional
+
+  * Category_name: `keyword-parameters`
+  * Automatic fix: yes
+
+Some parameters for builtin functions in Skyalrk are keyword for legacy reasons,
+their names are not meaningful (e.g. `x`), making them positional will improve
```

Comment:
The two commas in this sentence are ungrammatical. 
Replace the first with ";" or "and", and the second with a period and capital.

Reply:
Done.